api/server/sdk/volume_node_ops.go

```diff
@@ -89,7 +89,8 @@ func (s *VolumeServer) Detach(
 	}
 
 	// Get access rights
-	if err := s.checkAccessForVolumeId(ctx, req.GetVolumeId(), api.Ownership_Write); err != nil {
+	err := s.checkAccessForVolumeId(ctx, req.GetVolumeId(), api.Ownership_Write)
```

Comment:
Suggestion: You may want to pull in the implementation of `checkAccessForVolumeId` in case the implementation changes.

If you do not want to do it now, just add an issue to openstorage and/or a comment on that function of the expected return values.

Reply:
added function comment